RELEASING.md

```diff
@@ -0,0 +1,189 @@
+# Releasing
+
+The following steps outline the release process for both new minor versions (e.g.
+releasing the `master` branch as X.Y.0) and patch versions (e.g. releasing the
+`vX.Y` branch as X.Y.Z).
```

Comment:
Current branches use a naming strategy without a leading `v` (i.e. `3.9`). In the interest of consistency, I'd change this to `v4.0` starting with the 4.0 release series.

Reply:
IIRC, "v" prefixing was used to clearly distinguish branches from tags. I assume the ".x" suffix used by Laravel and Doctrine accomplishes the same thing.

That said, our tags always consist of three parts (e.g. `X.Y.Z`), so the real benefit here might just be removing the need to context switch between our various repositories.

Reply:
Okay, I guess it doesn't really matter. Consistency across our projects is the most important thing.

Reply:
Correct, that was the main motivation to do this.